src/Compilers/Core/Portable/Operations/ControlFlowGraphBuilder.cs

```diff
@@ -3878,6 +3885,69 @@ public override IOperation VisitTypeOf(ITypeOfOperation operation, int? captureI
             return new TypeOfExpression(operation.TypeOperand, semanticModel: null, operation.Syntax, operation.Type, operation.ConstantValue, operation.IsImplicit);
         }
 
+        public override IOperation VisitParameterInitializer(IParameterInitializerOperation operation, int? captureIdForResult)
+        {
+            var parameterRef = new ParameterReferenceExpression(operation.Parameter, semanticModel: null,
+                operation.Syntax, operation.Parameter.Type, constantValue: default, isImplicit: true);
+            VisitInitializer(rewrittenTarget: parameterRef, initializer: operation);
+            return null;
+        }
+
+        public override IOperation VisitFieldInitializer(IFieldInitializerOperation operation, int? captureIdForResult)
+        {
+            foreach (IFieldSymbol fieldSymbol in operation.InitializedFields)
+            {
+                IInstanceReferenceOperation instance = fieldSymbol.IsStatic ?
+                    null :
+                    new InstanceReferenceExpression(InstanceReferenceKind.ContainingTypeInstance, semanticModel: null,
+                        operation.Syntax, fieldSymbol.ContainingType, constantValue: default, isImplicit: true);
+                Optional<object> constantValue = fieldSymbol.HasConstantValue ? new Optional<object>(fieldSymbol.ConstantValue) : default;
+                var fieldRef = new FieldReferenceExpression(fieldSymbol, isDeclaration: false, instance, semanticModel: null,
+                    operation.Syntax, fieldSymbol.Type, constantValue, isImplicit: true);
+                VisitInitializer(rewrittenTarget: fieldRef, initializer: operation);
+            }
+
+            return null;
+        }
+
+        public override IOperation VisitPropertyInitializer(IPropertyInitializerOperation operation, int? captureIdForResult)
+        {
+            foreach (IPropertySymbol propertySymbol in operation.InitializedProperties)
+            {
+                var instance = propertySymbol.IsStatic ?
+                    null :
+                    new InstanceReferenceExpression(InstanceReferenceKind.ContainingTypeInstance, semanticModel: null,
+                        operation.Syntax, propertySymbol.ContainingType, constantValue: default, isImplicit: true);
+                var propertyRef = new PropertyReferenceExpression(propertySymbol, instance, arguments: ImmutableArray<IArgumentOperation>.Empty,
+                    semanticModel: null, operation.Syntax, operation.Type, constantValue: default, isImplicit: true);
```

Comment:
>operation.Type [](start = 59, length = 14)

Why type from the operation rather than from the property?

Reply:
Ah, that's an oversight, causing the baseline issues for property tests. I will this and the baselines. Thanks.

---
In reply to: [187450751](https://github.com/dotnet/roslyn/pull/26763#discussion_r187450751) [](ancestors = 187450751)